src/hooks/useCart/useCart.test.tsx

```diff
@@ -656,6 +726,67 @@ describe("useCart", () => {
       ]);
     });
 
+    it("should show PENDING_CONFIRMATION cart state when a PAYMENT_RECEIPT identifier input exists with whitespaced values", async () => {
+      expect.assertions(2);
+      const ids = ["ID1"];
+      const { result } = renderHook(
+        () => useCart(ids, key, endpoint, mockQuotaResSingleId.remainingQuota),
+        {
+          wrapper,
+        }
+      );
+
+      await waitFor(() => {
+        result.current.updateCart("toilet-paper", 1, [
```

Comment:
do you want to add the "chocolate" categories here with whitespaced values? so that it wont be empty for results below

Reply:
added!